public/web/WebView.h

```diff
@@ -375,8 +375,10 @@ class WebView : public WebWidget {
 
     // Accessibility -------------------------------------------------------
 
+#ifndef DISABLE_ACCESSIBILITY
     // Returns the accessibility object for this view.
     virtual WebAXObject accessibilityObject() = 0;
```

Comment:
The accessibilityObject interface is exported out at WebDocument and WebNode. How about delete the definations in above 4 files (.h and .cpp) , so we can found all the errors from compiling time.